src/widget/wknob.cpp

```diff
@@ -40,3 +44,18 @@ void WKnob::mouseReleaseEvent(QMouseEvent* e) {
 void WKnob::wheelEvent(QWheelEvent* e) {
     m_handler.wheelEvent(this, e);
 }
+
+void WKnob::guiTick() {
+    mixxx::Duration now = mixxx::Time::elapsed();
+    if (now - m_lastActivity > mixxx::Duration::fromSeconds(1)) {
+        m_guiTickTimer.stop();
+    }
+    update();
+}
+
```

Comment:
> It took me a few minutes to understand that GuiTick and GuiTickTimer were different classes and don't do the same jobs. I think it would help to rename GuiTick. My first suggestion is renaming GuiTick to "GuiTickProcessor", but maybe you can suggest a more descriptive name.

Maybe we can improve the name in master. I believe that'll cause merge conflicts for me in #941 and it seems low priority.

Reply:
Sure

Reply:
Eh oops, because GitHub put your response to my response to your top-level comment in this sub-thread, I thought I handled it. I'll add some more comments.